src/controllers/requestController.ts

```diff
@@ -95,6 +123,138 @@ export class RequestController {
         await this.runCore(httpRequest);
     }
 
+    @trace('Request Script')
+    public async runScript(range: Range) {
+        try {
+            const SCRIPT_EDITOR = window.activeTextEditor;
+            if (!SCRIPT_EDITOR || !SCRIPT_EDITOR.document) {
+                return;
+            }
+
+            const WORKSPACES = workspace.workspaceFolders;
+            const REQUEST_DOCUMENTS: StringDocument[] = [];
+
+            if (WORKSPACES) {
+                const FILES: string[] = [];
+                for (const WS of WORKSPACES) {
+                    (await glob('**/*.{http,rest}', {

```

Comment:
would it make sense to read the regex `.{http,rest}` from the `package.json` file https://github.com/Huachao/vscode-restclient/blob/master/package.json#L59?